students/sbolsen/lesson04/mailroom2.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import time
+
+donors = {
+        'Jesse Johnson': [150, 345],
+        'Mary May': [1000, 750],
+        'Spencer Samuels': [50, 200, 1100],
+        'Zach Zillow': [85],
+        'Tina Thompson': [76, 250, 300]
+        }
+
+
+def donor_list():
+    for donor in donors:
+        print(donor)
+
+
+def prompt_user():
+    choice = input('Please select from the following: \n1) Send a Thank You \n2) Create a Report \n3) Send letters to everyone \n4) Quit\n')
+    return choice
+
+
+def thank_you():
+    select_user = input('Enter a full name: ')
+    if select_user == 'list':
+        donor_list()
+        select_user = input('Enter a full name: ')
+    return select_user
+
+
+def user():
+    select_user = thank_you()
+    update_donation = int(input('Enter a donation amount: '))
+    try:
+        if select_user in donors.keys():
```

Comment:
The `in` operator checks for key membership in dict, so no need to call `keys()` here